cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -79,6 +79,29 @@ get_home() {
         getent passwd "$(id -un)" | cut -d: -f6
     fi
 }
+
+# Check if running on Windows with POSIX-compliant shell (CYGWIN, MSYS, MINGW)
+is_windows_posix() {
+    case "$(uname)" in
+        CYGWIN*|MSYS*|MINGW*)
+            return 0
+            ;;
+        *)
+            return 1
+            ;;
+    esac
+}
+# Convert POSIX path to Windows path
+convert_path_for_receipt() {
+    local _path="$1"
+    if is_windows_posix && command -v cygpath > /dev/null 2>&1; then
```

Comment:
We've already got a helper in this script for checking for the presence of commands: `check_cmd`. https://github.com/axodotdev/cargo-dist/blob/c53b7d91e9daf36bbedf8a37a60a3e9863142862/cargo-dist/templates/installer/installer.sh.j2#L1311C1-L1314